libgap.pc.in

```diff
@@ -0,0 +1,12 @@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@prefix@/include
+
+Name: GAP
+Description: Library containing the kernel of GAP, a computer algebra system
+URL: https://www.gap-system.org
+Version: @PACKAGE_VERSION@
```

Comment:
```suggestion
Version: @GAP_VERSION@
```
Just realized that `PACKAGE_VERSION` is an old left-over and should not actually be used here or anywhere.